stablehlo/tests/interpret_exponential.mlir

```diff
@@ -0,0 +1,13 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: exponential_op_test_i64
+func.func @exponential_op_test_i64() -> tensor<4xf64> {
```

Comment:
Can you add a test for `complex<f64>` element type?

Reply:
Sure, also changed names from %0 %1 to %operand %result